atlp-rwanda/e-comm-team-techsmith-fn
# #185517557 Bug fix implementing feedbacks from the predemo

## What does this PR do?

This PR applies the latest feedback from the predemo of 30th June, 2023.

## Description of Task to be completed?

- Redesign the wishlist page to enhance user experience and make it more intuitive.
- Add a wishlist icon (heart icon) in the navbar for easy navigation to the wishlist, applicable only to buyers who use the wishlist feature.
- Redesign the quantity input in the checkout form to improve visibility and usability.
- Ensure smooth transitions for the Modal in the checkout process.
- Prevent users from sending empty chats; disable the send button if the chat input is empty.

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-predemo-185517557`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

- [#185517557](https://www.pivotaltracker.com/story/show/185517557)
